src/Database/MorphPivot.php

```diff
@@ -0,0 +1,185 @@
+<?php namespace Winter\Storm\Database;
+
+class MorphPivot extends Pivot
```

Comment:
```suggestion
/**
 * Reimplementation of \Illuminate\Database\Eloquent\Relations\MorphPivot
 * but extends Winter's Pivot base class instead of Laravel's to get access to
 * Winter magic.
 */
class MorphPivot extends Pivot
```